outlines/generate/api.py

```diff
@@ -101,7 +101,7 @@ def strip_stop_sequences(
 
         return sequence
 
-    def format_sequence(self, sequence: str) -> str:
+    def format_sequence(self, sequence: str) -> Any:
```

Comment:
Is it possible to avoid using `Any`?

Reply:
It is, yep! It would just be a big union, due to all the different type casts, but I can add that instead.

Reply:
Changed this now.